VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/court-rooms-filters/court-rooms-filters.component.scss

```diff
@@ -0,0 +1,71 @@
+@import '~govuk-frontend/govuk/all';
+
+.vh-popup-overlay {
+  cursor: pointer;
+  height: 100%;
+  position: fixed;
+  text-align: center;
+  top: 0px;
+  width: 100%;
+  z-index: 10000;
+  float: left;
+  padding: 0px;
+  margin-left: 0px;
+}
+
+.vh-popup-overlay .helper {
+  display: inline-block;
+  height: 100%;
+}
+
+.vh-popup-overlay > div {
+  background-color: #fff;
+  box-shadow: 5px 10px 30px #555;
+  display: inline-block;
+  height: auto;
+  max-width: 650px;
```

Comment:
please avoid using pixels and use the vw or vh so this can scale accordingly across different machines

Reply:
done